tests/test_data/elife_poa_e14997.xml

```diff
@@ -1 +1 @@
/kwd></kwd-group><funding-group><award-group id="par-1"><funding-source><institution-wrap><institution-id institution-id-type="FundRef">http://dx.doi.org/10.13039/100000011</institution-id><institution>Howard Hughes Medical Institute</institution></institution-wrap></funding-source><principal-award-recipient><name><surname>Cembrowski</surname><given-names>Mark S</given-names></name><name><surname>Wang</surname><given-names>Lihua</given-names></name><name><surname>Sugino</surname><given-names>Ken</given-names></name><name><surname>Shields</surname><given-names>Brenda C</given-names></name><name><surname>Spruston</surname><given-names>Nelson</given-names></name></principal-award-recipient></award-group><funding-statement>The funders had no role in study design, data collection and interpretation, or the decision to submit the work for publication.</funding-statement></funding-group></article-meta></front><back><sec id="s1" sec-type="additional-information"><title>Additional information</title><fn-group content-type="competing-interest"><title>Competing interest</title><fn fn-type="conflict" id="conf1"><p>The authors declare that no competing interests exist.</p></fn></fn-group><fn-group content-type="ethics-information"><title>Ethics</title><fn fn-type="other"><p>Animal experimentation: Experimental procedures were approved by the Institutional Animal Care and Use Committee at the Janelia Research Campus (protocol #14-118).</p></fn></fn-group></sec><sec id="s2" sec-type="supplementary-material"><title>Additional Files</title><sec id="s3" sec-type="data-availability"><title>Data availability</title><p>Data Availability text <italic>here</italic> &amp; such</p><p>The following datasets were generated:</p><p><element-citation id="dataset1" publication-type="data" specific-use="isSupplementedBy"><person-group person-group-type="author"><collab>Cembrowski M</collab><collab>Spruston N</collab></person-group><year iso-8601-date="2016">2016</year><source>Hipposeq: an RNA-seq based atlas of gene expression in excitatory hippocampal neurons</source><ext-link ext-link-type="uri" xlink:href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?token=adsveykeprejbej&amp;acc=GSE74985">http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?token=adsveykeprejbej&amp;acc=GSE74985</ext-link><comment>GSE74985</comment></element-citation></p><p>The following previously published datasets were used:</p><p><element-citation id="dataset2" publication-type="data" specific-use="references"><person-group person-group-type="author"><collab>Cembrowski M</collab><collab>Spruston N</collab></person-group><year iso-8601-date="2016">2016</year><source>Spatial gene expression gradients underlie prominent heterogeneity of CA1 pyramidal neurons</source><ext-link ext-link-type="uri" xlink:href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67403">http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67403</ext-link><comment>GSE67403</comment></element-citation></p></sec></sec></back></article>
```

Comment:
not sure how this is compared with the result produced by (char by char, or XML tree to XML tree), and I remember an eLife internal standard want XMLs to be all on one line. Would it be possible to maintain this fixture as a document organized on multiple lines (e.g. `xmllint -format`) formatting?

Reply:
My understanding is whether these XML fixtures can be prettified - considering this particular repo will be archived and read-only as soon as it is possible to integrate it (the next few weeks I hope), something like that would probably be something we would try in the newer https://github.com/elifesciences/jats-generator library.

The object's ``output_xml()`` method (https://github.com/elifesciences/jats-generator/blob/develop/jatsgenerator/generate.py#L761) supports creating pretty output. We do not do that currently in production, however we could possibly use pretty files as fixtures and when generating test output for comparison. For coverage, I'd want to invoke at least one non-pretty output to make sure the logic for that isn't broken.